CHANGELOG.md

```diff
@@ -1,46 +1,30 @@
-## [0.4.10](https://github.com/software-mansion-labs/react-native-ci-cli/compare/v0.4.9...v0.4.10) (2024-08-27)
-
+## [0.4.10](https://github.com/software-mansion/setup-ci/compare/v0.4.9...v0.4.10) (2024-08-27)
 
 ### Bug Fixes
 
-* add credits and spell check ([#84](https://github.com/software-mansion-labs/react-native-ci-cli/issues/84)) ([91a00e2](https://github.com/software-mansion-labs/react-native-ci-cli/commit/91a00e2a24207ce7535acdb2bbd9d542d9d41479))
-
-
-
-## [0.4.9](https://github.com/software-mansion-labs/react-native-ci-cli/compare/v0.4.8...v0.4.9) (2024-08-27)
+- add credits and spell check ([#84](https://github.com/software-mansion/setup-ci/issues/84)) ([91a00e2](https://github.com/software-mansion-labs/react-native-ci-cli/commit/91a00e2a24207ce7535acdb2bbd9d542d9d41479))
 
+## [0.4.9](https://github.com/software-mansion/setup-ci/compare/v0.4.8...v0.4.9) (2024-08-27)
 
 ### Bug Fixes
 
-* add contents read permissions to detox workflows ([#83](https://github.com/software-mansion-labs/react-native-ci-cli/issues/83)) ([f6ab340](https://github.com/software-mansion-labs/react-native-ci-cli/commit/f6ab340ef66cda33727044e700c05a19f6ba2af9))
-
-
-
-## [0.4.8](https://github.com/software-mansion-labs/react-native-ci-cli/compare/v0.4.7...v0.4.8) (2024-08-27)
+- add contents read permissions to detox workflows ([#83](https://github.com/software-mansion/setup-ci/issues/83)) ([f6ab340](https://github.com/software-mansion-labs/react-native-ci-cli/commit/f6ab340ef66cda33727044e700c05a19f6ba2af9))
 
+## [0.4.8](https://github.com/software-mansion/setup-ci/compare/v0.4.7...v0.4.8) (2024-08-27)
 
 ### Bug Fixes
 
-* add check read permissions to detox workflows ([#82](https://github.com/software-mansion-labs/react-native-ci-cli/issues/82)) ([6aa5079](https://github.com/software-mansion-labs/react-native-ci-cli/commit/6aa5079824fd0b5612b59150346ac0c6f564d564))
-
-
-
-## [0.4.7](https://github.com/software-mansion-labs/react-native-ci-cli/compare/v0.4.6...v0.4.7) (2024-08-26)
+- add check read permissions to detox workflows ([#82](https://github.com/software-mansion/setup-ci/issues/82)) ([6aa5079](https://github.com/software-mansion-labs/react-native-ci-cli/commit/6aa5079824fd0b5612b59150346ac0c6f564d564))
```

Comment:
Commits need to be updated too

Reply:
actually, the commits SHA have not changed 😮 apparently they were not copied when moving repository

Reply:
Ok, I meant links (they still point to software-mansion-labs/react-native-ci-cli - you fixed issue link, but not the commit link), cause when I tried it did not work, but it seems to work now...

Reply:
oh right, forgot the `/g` when sedding

Reply:
fixed in 5c9aa40e56ece7d24676ed252e0eca7cd7aeb98a